config/dashboard.toml

```diff
@@ -66,7 +66,7 @@ dev_intelligent_routing_v2=false
 dev_modularity_v2=false
 dev_alt_payment_methods=false
 dev_hypersense_v2_product=false
-maintainence_alert=""
```

Comment:
This seems to be how it's handled in the Control Center repo, would need to be addressed there as well.

Reply:
[here](https://github.com/juspay/hyperswitch-control-center/blob/6572126fc65b9cdb1928f401b02347912cef2a3b/config/config.toml#L25), right?

Reply:
Not just the TOML file, also wherever it's being read in their code.

Reply:
have addressed that [here](https://github.com/juspay/hyperswitch-control-center/pull/2986). but i'm afraid if this change would start breaking anything or not. @JeevaRamu0104, can you please confirm this once?

Reply:
changes from sdk: https://github.com/juspay/hyperswitch-web/pull/1023